paths/node-runner/README.md

```diff
@@ -42,11 +42,40 @@ If you're determined to learn by doing, we recommend using [Node Pilot](https://
 
 ### Hardware
 
-**Hardware Requirements:** 4 CPU’s (or vCPU’s) | 8 GB RAM | 200GB Disk
+**Hardware Requirements:** 4 CPUs (or vCPUs) | 8 GB RAM | 200GB Disk
+
+Plan hardware for 4 vCPUs per Pocket node, but when creating the node allocate 16. This will allow nodes extra resources to close app large sessions.
+
+Since the Pocket node is a continuously running blockchain, new data is always being stored on it. As 200GB might suffice only for about six months, consider having much more storage than minimally required.  
```

Comment:
`Since the Pocket node is a continuously running blockchain, new data is always being stored on it.` -> `Since the Pocket node is a continuously synching the blockchain, new data is always being written to disk.`